docs/environment-setup.md

```diff
@@ -183,3 +189,6 @@ Run `npm run develop`
 
 If using default settings, a front-end should now be available:
 Open `localhost:8000`
+
+\_Note: If login function is required, `npm run build` must be used instead of `npm run develop`. For more information on test accounts to log into Telescope for development, please refer to section 5 of our [Login Document](login.md):
```

Comment:
**Note**: If login function is required, `npm run build` must be used instead of `npm run develop`. You will need to run `npm run build` every time you change your frontend code because login only works on our proper domains/origins. Our security model is pegged to the server origin. For more information on test accounts to log into Telescope for development, please refer to section 5 of our [Login Document](login.md):